client/client.go

```diff
@@ -293,3 +295,38 @@ func (lc *LogClient) postAndParse(path string, body []byte, rsp interface{}) (*H
 	}
 	return httpData, nil
 }
+
+// AddChain performs an add-chain request, posting the provided certificate
+// chain to the CT Log hosted at LogClient.url.  The first certificate in chain
+// should be the end-entity certificate, with the second chaining to the first
+// and so on to the last, which should either be the root certificate or a
```

Comment:
(nit) the root certificate   ...  this is Monologues root cert, right?

In that case the following sentence seems unnecessary, since for Monologue to submit to a log, it must have it's root in the log's root set.

Or might monologue submit chains that lead upto a log-specific intermediate CA cert?

---

I think my comments are about the fact the doc here reads rather generally (like it's a LogClient API for any application) when I would think it is only intended for use in Monologue, isn't it?